UnityProject/Assets/Scripts/Systems/Explosions/Explosion.cs

```diff
@@ -310,5 +169,26 @@ static void drawCircle(ExplosionData explosionData, int xc, int yc, int x, int y
 			explosionData.CircleCircumference.Add(new Vector2Int(xc + y, yc - x));
 			explosionData.CircleCircumference.Add(new Vector2Int(xc - y, yc - x));
 		}
+
+		private static int GetDistanceFromStrength(float strength)
+		{
+			return strength switch
```

Comment:
seems like an equation would fit this better

Reply:
suggest and equation

Reply:
https://www.desmos.com/calculator